### ISSUE TITLE
"fa-inc" seems to be the only one that works

### ISSUE THREAD

**Issue by InstanceOfMichael** [kevdotbadger/atom-font-awesome#4]
It's also the only one I'll never use.

**Comment by bennyp123** [comment]
I've got this issue as well

**Comment by RicardoAbad** [comment]
That's right. The usefull here must be each icon, not the link-tag. The link is writted only once. We think the interesting here is a snippets for each icon....
